Keda.Cosmosdb.Scaler/src/Services/CosmosDBLease.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Keda.CosmosDB.Scaler.Services
+{
+    public class CosmosDBLease
+    {
+        public string LeasesCosmosDBConnectionString { get; internal set; }
+        public string LeaseDatabaseName { get; internal set; }
+        public string LeaseCollectionName { get; internal set; }

```

Comment:
Nit - The class is called CosmosDBLease, what if it gets renamed to this:
```suggestion
        public string ConnectionString { get; internal set; }
        public string DatabaseName { get; internal set; }
        public string CollectionName { get; internal set; }

```